src/content/en/2019/pwa.md

```diff
@@ -206,3 +213,27 @@ driven by its inclusion as a feature in a number of popular JavaScript framework
 The HTTP Archive shows that, out of the total population of sites that register a service worker, 
 12.71% of them are using at least one of the Workbox libraries. This percentage is roughly consistent 
 across desktop and mobile, with a slightly lower percentage (11.46%) on mobile compared to desktop (14.36%).
+
+## Conclusion
+
+The stats in this chapter show that PWAs are still only used by a small percentage of sites. However this relatively small usage
+is driven by the more popular sites which have a much larger share of traffic, and pages beyond the home page may use this more:
+we showed that 15% of page loads use a service workers.
+
+PWAs have also often been seen as Google-driven technology and while the other browsers still have some way to go to catch up on
+it's support it's positive to see support becoming more widespread. [Maximiliano Firtman](https://twitter.com/firt) does a great
+job of tracking this, including
+[explaining Safari PWA support](https://medium.com/@firt/iphone-11-ipados-and-ios-13-for-pwas-and-web-development-5d5d9071cc49)
+in light of poor documentation from Apple on that front. He (amongst others) also questions whether the tech giants really want
```

Comment:
OK toned this down again a little too. Personally I do think the App Store issue is an interesting one that should be covered in the chapter. But happy to remove it if you still don't like the revised wording. As you say, it will be your name against it so important you're comfortable with it.

Either way let me know your thoughts.